active/0000-ember-install.md

```diff
@@ -0,0 +1,35 @@
+- Start Date: (fill me in with today's date, YYYY-MM-DD)
+- RFC PR: (leave this empty)
+
+# Summary
+
+`ember install` without arguments should install all packages.
+`ember install package-name` should not error if the package is not an
+ember addon.
+
+# Motivation
+
+Since Ember CLI can now decide for you whether to use Yarn or not, we would like to
+take advantage of that in more ways.
+
+# Detailed design
```

Comment:
Wouldn't this just piggy-back on how `ember install package-name` finds the `ember` bin? Why would this RFC be focused on that resolution?